README.md

```diff
@@ -90,15 +90,15 @@ build.
 
 #### MinGW
 
-[MSYS2][msys2] can be used to easily build Rust on Windows:
+[MSYS2][msys2] can be used to build Rust on Windows easily:
```

Comment:
Hmm, you said in the PR description 'Increase readability for non-native speakers'. What makes this easier to read? As a native speaker I slightly prefer the first one (because 'easily' is closer to 'build'), but they're both pretty readable.

Reply:
This avoids having _easily_ split the infinitive. When I learned English this is something we were told to avoid.

See this [article](https://grammarbadgers.wordpress.com/teaching/esl-teaching-materials/split-infinitive/)

Reply:
Splitting infinitives is one of those things that some people do disagree on. I think this change is pretty value-neutral.